.azure-pipelines/client.yml

```diff
@@ -63,10 +63,13 @@ jobs:
       matrix:
         Linux:
           OSVmImage: 'ubuntu-16.04'
+          OSName: 'Linux'
         Windows:
           OSVmImage: 'vs2017-win2016'
+          OSName: 'Windows'
         MacOs:
           OSVmImage: 'macOS-10.13'
+          OSName: 'MacOS'
```

Comment:
nit: For consistency with all the other build definitions could you please have OSName before OSVmImage?